src/current/cockroachcloud/folders.md

```diff
@@ -0,0 +1,390 @@
+
+    {{site.data.alerts.callout_info}}
+    A cluster cannot be renamed.
+    {{site.data.alerts.end}}
+
+    A user with the Org Administrator or Folder Admin role can grant another user or a service account the Folder Mover role. Because the Folder Admin role is a superset of Folder Mover, there is no need for a Folder Admin to grant themselves the Folder Mover role.
+
+The remainder of this page shows how to create folders, manage access to them, and use them to organize your clusters.
+
+## Initial setup
+
+### Gather required information
+
+Your user account must have the following roles to create a service account and grant roles at the organization scope:
+
+- [Org Administrator](authorization.html#org-administrator)
+
+1. Sign in to the {{ site.data.products.db }} Console.
+
+1. From the **Organization** **Information** page, make a note of your organization's ID.
+
+1. [Create a service account](managing-access.html#create-a-service-account) and grant it the Organization Admin role. When prompted, create an API key for the service account and make a note of it. In the service account's **Details** page, make a note of the service account's ID, which is the portion of the URL after the last `/` character.
+
+    This service account will be referred to as the _provisioning service account_ in the remainder of this page.
+
+1. Create a second service account and grant it the Cluster Admin role. This service account will be referred to as the folder-management service account. When prompted, create an API key for the service account and make a note of it. In the service account's **Details** page, make a note of the service account's ID, which is the portion of the URL after the last `/` character.
+
+    This service account will be referred to as the _folder-management service account_ in the remainder of this page.
+
+### Grant the `FOLDER_ADMIN` role to the service account
+
+Folders inherit roles granted at the organization scope, and descendent folders inherit roles granted higher in the hierarchy.
+
+During Limited Access, it is not possible to use the {{ site.data.products.db }} Console to grant the Folder Admin or Folder Mover roles.
+Instead, use the CockroachDB {{ site.data.products.db }} API and the provisioning service account you created in [Gather required information](#gather-required-information) to grant the folder-management service account the `FOLDER_ADMIN` role at the level of the organization.
+
+1. To grant the role at the level of the organization:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    curl --request POST \
+           --url 'https://cockroachlabs.cloud/api/v1/roles/<folder-management-service-account-id>/ORGANIZATION/<organization_id>/FOLDER_ADMIN' \
+           --header 'Authorization: Bearer <provisioning_service_account_api-key>'
+    ~~~
+
+    Replace:
+    - `<folder-management-service-account-id>`: the ID of the user or service account being granted the role.
```

Comment:
I would suggest the following here:

- `<service_account_id>`: the service account ID (the role grantee)
- `<organization_id>`:your organization's ID.
- `<api_key>`: the role granter's API Key (not the API Key of the role grantee)

Reply:
Done